File: app/src/main/java/org/jellyfin/androidtv/startup/SelectServerActivity.java
package org.jellyfin.androidtv.startup;

import android.os.Bundle;

import org.jellyfin.androidtv.base.BaseActivity;

public class SelectServerActivity extends BaseActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        getSupportFragmentManager()
                .beginTransaction()
                .replace(android.R.id.content, new SelectServerFragment())
                .commit();
    }
}

File: app/src/main/java/org/jellyfin/androidtv/startup/SelectServerActivity.kt
package org.jellyfin.androidtv.startup

import android.os.Bundle
import org.jellyfin.androidtv.base.BaseActivity

class SelectServerActivity : BaseActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)

		supportFragmentManager
			.beginTransaction()
			.replace(android.R.id.content, SelectServerFragment())
			.commit()
	}
}
